src/main/java/racingcar/RacingCarGame.java

```diff
@@ -0,0 +1,88 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingCarGame {
+    private static final int LIMIT_NAME = 5;
+    private static final int RANDOM_SCOPE = 10;
+    private final List<Car> carList = new ArrayList<>();
+
+    public void start(String text, int racingNum) {
+
+        initCar(splitName(text));
+
+        printResult();
+        for (int i = 0; i < racingNum; i++) {
+            racingGame();
+            printResult();
+        }
+    }
+
+    private void initCar(String[] names) {
+        for (String name : names) {
+            carList.add(new Car(name.trim()));
+        }
+    }
+
+    private int randomNum() {
+        return (int) (Math.random() * RANDOM_SCOPE);
+    }
+
+    private void racingGame() {
+        for (Car car : carList) {
+            car.racing(randomNum());
+        }
+    }
+
+    private void printResult() {
+        for (Car car : carList) {
+            System.out.println(car);
+        }
+        System.out.println();
+    }
+
+    public String[] splitName(String text) {
+        return checkName(text.split(","));
+    }
+
+    private String[] checkName(String[] names) {
+        for (String name : names) {
+            checkOverflow(name);
+        }
+        return names;
+    }
+
+    private void checkOverflow(String name) {
+        if (name.length() > LIMIT_NAME) {
+            throw new RuntimeException();
+        }
+    }
+
+    private int getMaxPosition() {
+        int mx = 0;
+        for (Car car : carList) {
+            mx = Math.max(mx, car.getPosition());
+        }
+        return mx;
```

Comment:
현재 프로젝트가 java 1.8 기준이니 stream을 써보는건 어떨까요? 예시를 들면 아래와 같은데요. 
다른 로직들도 for문과 내부적으로 조건문을 쓴다면 java8의 stream 으로 변경이 가능해 보입니다.
```suggestion
return carList.stream()
                .mapToInt(Car::getPosition)
                .max()
                .getAsInt();
```

Reply:
네 제가 stream을 안 써봐서 떠올리지 못했던 것 같습니다. stream을 공부해서 여러가지를 적용해보겠습니다. 감사합니다.

Reply:
네 이번에 꼭 변경하실 필요는 없습니다. 다만 좀더 간결해지는 부분도 있고, stream을 이용하면 장점도 분명 있어서 추천드려요.